### PR TITLE
feat(ids): Audkenni Version 2 (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by Snojo**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: charts/identity-server/values.prod.yaml [hunk 1]
```diff
@@ -76,6 +76,7 @@ identity-server:
     AWS__SystemsManager__ParameterStore__DataProtectionPrefix: '/k8s/identity-server/DataProtectionSecret'
     Application__MinCompletionPortThreads: '10'
     AudkenniSettings__Retries: '24'
+    AudkenniSettings__Url: 'https://tqd75.audkenni.is:443/sso/'
```
Comment (valurefugl) [inline charts/identity-server/values.prod.yaml:79]: Do we want to modify these values for prod?

File: charts/identity-server/values.staging.yaml [hunk 2]
```diff
@@ -77,6 +77,7 @@ identity-server:
     AWS__SystemsManager__ParameterStore__DataProtectionPrefix: '/k8s/identity-server/DataProtectionSecret'
     Application__MinCompletionPortThreads: '10'
     AudkenniSettings__Retries: '24'
+    AudkenniSettings__Url: 'https://tqd75.audkenni.is:443/sso/'
```
Comment (valurefugl) [inline charts/identity-server/values.staging.yaml:80]: Same for staging.

**Review by valurefugl — CHANGES_REQUESTED** [review]

File: charts/identity-server/values.prod.yaml (hunk 1 above)
Comment (pshomov) [reply charts/identity-server/values.prod.yaml:79]: This PR will not be merged as is to prod. It is just for the feature deployment for now.

**Comment by pshomov** [comment]
So drop this PR and delete the branch?

**Pull request closed without merging**